packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -293,7 +293,26 @@ export class Popover
     this.setFilteredPlacements();
     connectLocalized(this);
     connectMessages(this);
-    connectFocusTrap(this);
+    connectFocusTrap(this, {
+      focusTrapEl: this.el,
+      focusTrapOptions: {
+        allowOutsideClick: true,
+        clickOutsideDeactivates: (event: MouseEvent) => {
```

Comment:
nitpick: what about making these anonymous functions private methods on the component? Applies to all the `clickOutsideDeactivates` options that are functions.

May also be beneficial for `onDeactivate` too.